### DISCUSSION TITLE
Support for Astra DB

### DISCUSSION THREAD

**Discussion by roi-tevel** [JanusGraph/janusgraph#3757] [Q&A]
Hi, in the documentation I see that one can deploy JanusGraph with AstraDB https://docs.janusgraph.org/storage-backend/cassandra/#deploying-on-datastax-astra, but I don't see AstraDB in the list of tested compatibilities. do you recommend deploying with AstraDB in production? is it tested? will backup/restore of data work? and will JanusGraph respect these AstraDB limitations? https://docs.datastax.com/en/astra-serverless/docs/plan/planning.html#_columns

Thanks, 
Roi

**Comment by porunov** [comment]
JanusGraph doesn't list AstraDB in tested compatibilities because we don't have direct access to AstraDB to run out test suite against this database. 
AstraDB documentation contribution was initiated by AstraDB team who tested this compatibility.
In addition to this, I can confirm that JanusGraph 1.1.0 works with AstraDB just fine. AstraDB have their own backup system that works as well.
